app/controllers/admin/demo_nights_controller.rb

```diff
@@ -1,4 +1,11 @@
 class Admin::DemoNightsController < Admin::BaseController
+  before_action :check_active_demo_nights, only: :create
+
+  def index
+    demo_nights = DemoNight.all
+    @inactives = demo_nights.where(status: "closed")
+    @currents = demo_nights.where.not(status: "closed")
```

Comment:
*8pm I'm exhausted thought:* pass `@demo_nights` to the view, and create two methods on DemoNight that we then use in the view:

```
def active_list
  where.not(status: "closed")
end

def inactive_list
  where.(status: "closed")
end
```

Other thoughts:
* Can those two methods be stabby lambda style scopes?
* Re: naming - `active?` is currently an instance method on DemoNight that checks a single DemoNight (seems to make sense), but I'd initially hoped to name the method above `active`, which would then be called on the class. This feels like I'm setting us up for confusion, hence the `active_list`. Is there a better way to name these?